### PR TITLE
Add docs for SDK + Widget usage

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: packages/node_modules/@ciscospark/widget-recents/README.md [hunk 1]
```diff
@@ -104,6 +104,14 @@ To use the space widget within an existing React appliction, a developer can ins
 
   All of the React configurable properties are listed in the ["Configuration"](#Configuration) section.
 
+#### Usage with [Webex SDK](https://developer.webex.com/docs/sdks/browser)
```
Comment (lalli-flores) [inline packages/node_modules/@ciscospark/widget-recents/README.md]: Is this something you want to add to the TOC?

Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-recents/README.md]: Done

File: packages/node_modules/@ciscospark/widget-recents/README.md [hunk 2]
```diff
@@ -104,6 +105,14 @@ To use the space widget within an existing React appliction, a developer can ins
 
   All of the React configurable properties are listed in the ["Configuration"](#Configuration) section.
 
+#### Usage with Webex SDK
+
+If you are developing an application that makes use of our [Webex SDK](https://developer.webex.com/docs/sdks/browser) in addition to the widgets, some additional configuration is needed.
+
+In order for the Webex Teams Recents Widget to function in a project that is already using the Webex SDK via npm, the versions of the SDK must match specifically.
+
+Any "@webex" packages that are added to your project manually need to match the versions in the widget's [package.json](https://github.com/webex/react-widgets/blob/master/package.json) file.
```
Comment (mattnorris) [inline packages/node_modules/@ciscospark/widget-recents/README.md]: widgets'

File: packages/node_modules/@ciscospark/widget-space/README.md [hunk 3]
```diff
@@ -136,6 +137,13 @@ To use the space widget within an existing React appliction, a developer can ins
     <SpaceWidget {...spaceWidgetProps} />
   ```
 
+#### Usage with Webex SDK
+
+If you are developing an application that makes use of our [Webex SDK](https://developer.webex.com/docs/sdks/browser) in addition to the widgets, some additional configuration is needed.
+
+In order for the Webex Teams Space Widget to function in a project that is already using the Webex SDK via npm, the versions of the SDK must match specifically.
+
+Any "@webex" packages that are added to your project manually need to match the versions in the widget's [package.json](https://github.com/webex/react-widgets/blob/master/package.json) file.
```
Comment (mattnorris) [inline packages/node_modules/@ciscospark/widget-space/README.md]: widgets'

File: packages/node_modules/@ciscospark/widget-recents/README.md (hunk 2 above)
Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-recents/README.md]: done

File: packages/node_modules/@ciscospark/widget-space/README.md (hunk 3 above)
Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-space/README.md]: done

**Pull request merged**